rebar.config.script

```diff
@@ -1,21 +1,3 @@
 os:putenv("EXOMETER_PACKAGES", "(basic)"),
```

Comment:
do we need this `rebar.config.script` at all? why can't we remove it?

Reply:
I'll check that if we still need it. We needed it some time ago in order to prevent exometer from downloading all possible deps which we didn't need.